JHUGenerator/mod_Parameters.F90

```diff
@@ -3,7 +3,7 @@ MODULE ModParameters
 save
 !
 !
-character(len=*),parameter :: JHUGen_Version="v7.4.1"
+character(len=*),parameter :: JHUGen_Version="v7.5.0"
```

Comment:
No changes in JHUGen version are warranted since there is no actual code change.

Reply:
We are moving forward with the official version 7.5.0 as agreed long ago. 
This printout should appear in all printouts of the next public version.

Reply:
There is no such agreement, but either way, necessary steps are not yet taken to merge this PR.

Reply:
Of course there is an agreement for a long time. 
Merging is of course pending, but we can do this tomorrow.